packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/general.tsx

```diff
@@ -313,15 +314,49 @@ const NewDocDefaultModeSettings = () => {
   );
 };
 
+const AISettings = () => {
+  const { openConfirmModal } = useConfirmModal();
+  const { editorSettingService } = useServices({ EditorSettingService });
+
+  const settings = useLiveData(editorSettingService.editorSetting.settings$);
+
+  const onAIChange = useCallback(
+    (checked: boolean) => {
+      editorSettingService.editorSetting.set('enableAI', checked);
+      // refresh the page to apply the change
+      window.location.reload();
+    },
+    [editorSettingService.editorSetting]
+  );
+  const onToggleAI = useCallback(
+    (checked: boolean) => {
+      openConfirmModal({
+        title: checked ? 'Enable AI?' : 'Disable AI?',
+        description: `Are you sure you want to ${checked ? 'enable' : 'disable'} AI? This will refresh the page to apply the change.`,
```

Comment:
i18n ?

Reply:
The copy for this confirm modal is yet to be finalised. I will add the i18n once the copy is confirmed.